etl-pipeline/tests/conftest.py

```diff
@@ -448,3 +451,13 @@ def mock_sqs_manager():
         MockSQSManager.return_value = mock_sqs_manager_instance
 
         yield mock_sqs_manager_instance
+
+
+@pytest.fixture(scope="module")
+def grin_client():
+    client = GRINClient()
+    assert client is not None
+    assert isinstance(client.creds, Credentials)
+    assert client.session is not None
+    assert isinstance(client.session, AuthorizedSession)
+    yield client
```

Comment:
quick point on readability - i like the assertions here to make sure things are solidly setup in the client. I recommend to use new lines to logically break up the code. So below we are using the new lines to break instantiations, assertions, yield/return statement. 

```
client = GRINClient()

assert client is not None
assert isinstance(client.creds, Credentials)
assert client.session is not None
assert isinstance(client.session, AuthorizedSession)

yield client
```